target/linux/rockchip/patches-6.12/403-02-v6.13-arm64-dts-rockchip-Enable-HDMI0-on-rock-5a.patch

```diff
@@ -0,0 +1,90 @@
+From f57a8daf6bbd8e71f16693ad6d8421cb881c7fe0 Mon Sep 17 00:00:00 2001
```

Comment:
why do you add this then revert it in `403-98-arm64-dts-rockchip-Revert-dts-for-Radxa-ROCK-5A.patch`? please remove both

Reply:
The `hdmi*` nodes have been backported and unnecessary revert patches have been removed.